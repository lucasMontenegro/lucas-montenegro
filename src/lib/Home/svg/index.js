import { ReactComponent as LightOnlineProfile } from "./onlineProfile/light.svg"
import { ReactComponent as DarkOnlineProfile } from "./onlineProfile/dark.svg"
import { ReactComponent as LightTechnologies } from "./technologies/light.svg"
import { ReactComponent as DarkTechnologies } from "./technologies/dark.svg"
import { ReactComponent as LightWebsite } from "./website/light.svg"
import { ReactComponent as DarkWebsite } from "./website/dark.svg"
export default {
  onlineProfile: {
    light: LightOnlineProfile,
    dark: DarkOnlineProfile,
    aspectRatio: `46%`,
  },
  technologies: {
    light: LightTechnologies,
    dark: DarkTechnologies,
    aspectRatio: `48%`,
  },
  website: {
    light: LightWebsite,
    dark: DarkWebsite,
    aspectRatio: `74%`,
  },
}