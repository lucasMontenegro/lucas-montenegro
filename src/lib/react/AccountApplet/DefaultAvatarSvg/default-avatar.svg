<svg xmlns="http://www.w3.org/2000/svg" height="64" width="64" version="1.1" viewBox="0 0 21.166666 21.166667">
 <g transform="translate(0 -275.83)">
  <path style="fill:#424242;fill-opacity:.31765;paint-order:normal;fill-rule:evenodd" transform="matrix(.26458 0 0 .26458 0 275.83)" d="m0 0v80h40 40v-80h-40-40zm40 16a11.52 11.52 0 0 1 11.52 11.52 11.52 11.52 0 0 1 -11.52 11.521 11.52 11.52 0 0 1 -11.52 -11.521 11.52 11.52 0 0 1 11.52 -11.52zm-6 26.24h6 6a18 9.8401 0 0 1 18 9.84v9.359a2.56 2.56 0 0 1 -2.561 2.561h-42.878a2.56 2.56 0 0 1 -2.561 -2.561v-9.359a18 9.8401 0 0 1 18 -9.84z"/>
 </g>
</svg>
