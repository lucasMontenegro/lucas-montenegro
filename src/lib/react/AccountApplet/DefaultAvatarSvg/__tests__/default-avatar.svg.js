import renderer from "react-test-renderer"
import React from "react"
import { ReactComponent } from "../default-avatar.svg"
describe(`../default-avatar.svg`, () => {
  jestUtils.describeDependencies({
    deps: [
      `react`,
      `react-scripts`,
      `react-test-renderer`,
    ],
    relativeBasePath: __dirname,
  })
  describe(`<ReactComponent />`, () => {
    it(`should render`, () => {
      expect(renderer.create(<ReactComponent />).toJSON()).toMatchSnapshot()
    })
  })
})