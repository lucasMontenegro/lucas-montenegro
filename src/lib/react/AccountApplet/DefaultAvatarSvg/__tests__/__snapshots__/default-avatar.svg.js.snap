// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`../default-avatar.svg <ReactComponent /> should render 1`] = `
<svg>
  default-avatar.svg
</svg>
`;

exports[`../default-avatar.svg external dependencies react should match the installed version 1`] = `"^16.10.1"`;

exports[`../default-avatar.svg external dependencies react-scripts should match the installed version 1`] = `"^3.3.1"`;

exports[`../default-avatar.svg external dependencies react-test-renderer should match the installed version 1`] = `"^16.10.1"`;
